services/strimzi-registry-operator/Chart.yaml

```diff
@@ -1,7 +1,5 @@
 apiVersion: v2
```

Comment:
I'm confused; strimzi-registry-operator was added as a shared chart earlier. It shouldn't be both a service and a shared chart; it should be one or the other, no?